com.unity.cinemachine/Tests/Editor/ScriptableObjectUtilityTests.cs

```diff
@@ -10,15 +11,19 @@ public class ScriptableObjectUtilityTests
         [Test]
         public void CinemachineInstallPathIsValid()
         {
-            var pathToCmLogo = Path.Combine(ScriptableObjectUtility.CinemachineInstallPath, "Editor/EditorResources/cm_logo_sm.png");
```

Comment:
We should get rid of ScriptableObjectUtility.CinemachineInstallPath, it's no longer useful.  It dates from the days when Cinemachine could be installed in the Assets folder